subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df.c

```diff
@@ -207,7 +218,51 @@ void lll_df_conf_cte_rx_enable(uint8_t slot_duration, uint8_t ant_num, uint8_t *
 }
 #endif /* CONFIG_BT_CTLR_DF_SCAN_CTE_RX */
 
+/* @brief Function performs common steps for initialization and reset
+ * of Direction Finding LLL module.
+ *
+ * @return	Zero in case of success, other value in case of failure.
+ */
 static int init_reset(void)
 {
 	return 0;
 }
+
+#if defined(CONFIG_BT_CTLR_DF_ADV_CTE_TX)
+/* @brief Function initializes transmission of Constant Tone Extension.
+ *
+ * @param[in] type      Type of CTE: AoA, AoD 1us, AoD 2us
+ * @param[in] length    Duration of CTE in 8us units
+ * @param[in] ant_num   Number of antennas in switch pattern
+ * @param[in] ant_ids   Antenna identifiers that create switch pattern.
+ *
+ * @Note Pay attention that first two antenna identifiers in a switch
+ * pattern has special purpose. First one is used in guard period, second
+ * in reference period. Actual switching is processed from third antenna.
+ *
+ * In case of AoA mode ant_num and ant_ids parameters are not used.
+ */
+static void df_cte_tx_configure(uint8_t type, uint8_t length, uint8_t ant_num,
+				const uint8_t *ant_ids)
+{
+	if (type == BT_HCI_LE_AOA_CTE) {
+		radio_df_mode_set_aoa();
+		radio_df_cte_tx_aoa_set(length);
+	}
+#if defined(CONFIG_BT_CTLR_DF_ANT_SWITCH_TX)
```

Comment:
`radio_df_ant_switching_pin_sel_cfg` is dependent on both configs. The function has to be conditionally compiled because it depends on `ant_cfg` global variable that is also defined conditionally.